test/system/test_tpm2_nv.sh

```diff
@@ -134,7 +134,11 @@ fi
 large_file_name="nv.test_large_w"
 
 if [ ! -f $large_file_name ]; then
-  base64 /dev/urandom | head -c $large_file_size > $large_file_name
+  base64 /dev/urandom | head -c $(($large_file_size)) > $large_file_name
```

Comment:
How does the arithmetic style parens fix the issue?

Reply:
@williamcroberts  Right, I meant to mention in the commit message but then it got already to verbose :smiley: 

When using arithmetic expansions, the expressions are evaluated according to bash's arithmetic evaluation rules. And this says that if you don't define a base for a number, then decimal is used.

I can add a comment in the script if you think is too cryptic.

Reply:
Oh cool, so that actually converts hex to base 10 in bash.
```
$ foo=0x42
$ echo $((foo))
66
```